@here/harp-mapview/lib/MapView.ts

```diff
@@ -496,6 +496,14 @@ export interface MapViewOptions {
      */
     maxFps?: number;
 
+    /**
+     * Disable phased loading. If `true`, the geometry on a [[Tile]] is always being created in a
+     * single step, instead of (potentially) over multiple frames to smoothen animations.
+     *
+     * @default `undefined`
+     */
+    disablePhasedLoading?: boolean;
```

Comment:
It is usually better to have "enabled"-kind flags (defaulted to true in this case), it's simply to read and understand.
```suggestion
    enablePhasedLoading?: boolean;
```

Reply:
That is true, changing the wording. But your "suggested change" would not be sufficient since more changes are required if the logic/default value is not supposed to change.

Reply:
Yes, it's not sufficient. I just don't know how to suggest a couple of changes around files.

Reply:
You do not need to add suggestions here, the author will handle that. If the motivation for the requested change is substantiated and well explained (as is is here), your job is done! ;)

Reply:
Ok. Thanks for the advice!